modules/xfce/hm.nix

```
@@ -6,14 +6,37 @@
     config.lib.stylix.mkEnableTarget "Xfce" false;
 
   config = lib.mkIf (config.stylix.enable && config.stylix.targets.xfce.enable) {
-    xfconf.settings = with config.stylix.fonts; {
-      xfwm4 = {
+    xfconf.settings = {
+      xfwm4 = with config.stylix.fonts; {
         "general/title_font" = "${sansSerif.name} ${toString sizes.desktop}";
       };
-      xsettings = {
+      xsettings = with config.stylix.fonts; {
         "Gtk/FontName" = "${sansSerif.name} ${toString sizes.applications}";
         "Gtk/MonospaceFontName" = "${monospace.name} ${toString sizes.applications}";
       };
+      xfce4-terminal = with config.lib.stylix.colors.withHashtag; {
+        "color-palette" = lib.strings.concatStringsSep ";" [
+          base00 # Normal
+          base08
+          base0B
+          base0A
+          base0D
+          base0E
+          base0C
+          base05
+          base02 # Bright
```

I don't think we need the comments:

```suggestion
          base00
          base08
          base0B
          base0A
          base0D
          base0E
          base0C
          base05
          base02
```